unit test error on Non English Environments

**Describe the bug**

Unit Test had Error in StringCellParserTest.kt on Non English Environments.

because,  Dec(December) is only used in English.

In Spanish: "Dic" (diciembre)
In French: "Déc" (décembre)
In German: "Dez" (Dezember)
In Italian: "Dic" (dicembre)
In Korean: "12"

```
> Task :test

StringCellParserTest > should try to parse to string or return null if failed to do that() FAILED
    org.opentest4j.AssertionFailedError at StringCellParserTest.kt:30

StringCellParserTest > should parse to string() FAILED
    org.opentest4j.AssertionFailedError at StringCellParserTest.kt:18

67 tests completed, 2 failed

> Task :test FAILED

FAILURE: Build failed with an exception.
```

https://github.com/VorTECHsa/refinery/blob/8ca98ba2154dd7f9405d1cab7e805db907dd52bd/src/test/java/com/vortexa/refinery/cell/parser/StringCellParserTest.kt#L18

https://github.com/VorTECHsa/refinery/blob/8ca98ba2154dd7f9405d1cab7e805db907dd52bd/src/test/java/com/vortexa/refinery/cell/parser/StringCellParserTest.kt#L30


**Expected behaviour**
Unit Test was Passed.

**Actual behaviour**
Unit Test was Failure.

**Additional context**
Add any other context about the problem here.

--- a/src/test/java/com/vortexa/refinery/cell/parser/CellParserTest.kt
+++ b/src/test/java/com/vortexa/refinery/cell/parser/CellParserTest.kt
@@ -14,6 +14,7 @@ import org.junit.jupiter.api.BeforeAll
 import org.junit.jupiter.api.TestInstance
 import org.junit.jupiter.api.TestInstance.Lifecycle.PER_CLASS
 import java.io.File
+import java.util.Locale
 
 @TestInstance(PER_CLASS)
 open class CellParserTest {
@@ -29,6 +30,7 @@ open class CellParserTest {
 
         workbook = WorkbookFactory.create(file)
         sheet = workbook.getSheet("cell_parsers")
+        Locale.setDefault(Locale.ENGLISH)
     }
 
     @AfterAll

